packages/playwright-ct-react-17/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@playwright/experimental-ct-react-17",
+  "version": "1.31.0-next",
+  "description": "Playwright Component Testing for React 17 and earlier",
+  "repository": "github:Microsoft/playwright",
+  "homepage": "https://playwright.dev",
+  "engines": {
+    "node": ">=14"
+  },
+  "author": {
+    "name": "Microsoft Corporation"
+  },
+  "license": "Apache-2.0",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "default": "./index.js"
+    },
+    "./register": {
+      "types": "./register.d.ts",
+      "default": "./register.mjs"
+    },
+    "./hooks": {
+      "types": "./hooks.d.ts",
+      "default": "./hooks.mjs"
+    }
+  },
+  "dependencies": {
+    "@vitejs/plugin-react": "^3.0.0",
+    "@playwright/test": "1.31.0-next",
+    "vite": "^4.0.3"
+  }
```

This is also needed since https://github.com/microsoft/playwright/pull/20669:

```suggestion
  },
  "bin": {
    "playwright": "./cli.js"
  }
```
`cli.js` file:
```javascript
#!/usr/bin/env node
/**
 * Copyright (c) Microsoft Corporation.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
module.exports = require('playwright-core/cli');
```

---

done